demos/image_processing_demo/cpp/README.md

```diff
@@ -0,0 +1,116 @@
+# Image Processing C++ Demo
+
+This demo processes the image according to the selected type of processing. The demo can work with the next types:
```

Comment:
please look at other demos, we have recently modified demo's README template, a few new sections added

Reply:
added